oct/ansible/oct/playbooks/package/ami-mark-ready.yml

```diff
@@ -55,4 +55,13 @@
         resource: '{{ origin_ci_aws_ami_id }}'
         tags:
           ready: 'yes'
-        state: present
\ No newline at end of file
+        state: present
+
+    - name: add any additional tags
```

Comment:
Would be good to move `oct package ami --mark-ready` to a different command but that would require finding a new verb ... or making the `--mark-ready` be appropriate for all other runs